src/clarity-angular/buttons/_buttons.clarity.scss

```diff
@@ -215,6 +241,64 @@ $clr-button-appearance-map: (
         @include generateButton();
         @include clr-getStylesFromMap($clr-button-appearance-map, form);
         @include populateButtonProperties(default);
+    }
+
+    .btn-group {
+        //.btn-outline-<state> is the new BS4 naming convention
+        //TODO: we support both the previous one and the new one but we will deprecate the old ones from the example in 0.5.0
+        &.btn-secondary > .btn,
```

Comment:
I don't think that should apply to net-new components.

It doesn't apply to the wizard refactor, for example.

Nor does it apply to vertical tabs.

I'm fine leaving the _existing_ classes as they are. But what we are doing here is introducing NEW classes that are using an old, bad way of doing that.

I'm not okay with that.